src/Components/CardList/Card/CardHeader/index.jsx

```diff
@@ -1,26 +1,26 @@
 import React from 'react';
 import { AiFillEdit, AiOutlineCheck, AiOutlineClose } from "react-icons/ai";
+import { Route } from 'react-router-dom';
 
 const CardHeader = props => {
-
   const defaultMode = (
     <div className='card_title'>
       <p>{props.card.title}</p>
-      <div>
-        {!props.onView && <button title='режим редактирования' onClick={props.editModeOn}><AiFillEdit /></button>}
-        <input className='card_checbox' type='checkbox' onClick={props.selectItem} />
+      <div className='card_title__btn_block'>
+        <Route path='/' exact>
+          <input className='card_checbox' type='checkbox' onClick={props.selectItem} />
+        </Route>
```

Comment:
Is there any specific reason to add this route? I do not see any. You can pass a property saying of it's a card is viewed in a list or on a page and check it in order to render the checkbox.

Reply:
I decided that this function does not make sense on the "card /id" page.
Fixed